usr/src/psm/stand/boot/aarch64/common/arch_timer.c

```diff
@@ -0,0 +1,129 @@
+/*
+ * CDDL HEADER START
```

Comment:
You probably want the header from `usr/src/prototypes/prototype.c` not the old OpenSolaris one.

Reply:
Done.